### PR TITLE
Improve create/restore and UI loading landing page

### LINKED ISSUE #2: Open a "Landing" page when there are no wallets
### Can the feature be described as a User Story?
Given that there are no detected wallets on the system
When I launch the application
Then I should see a "Landing" page
With a "Create Wallet" option
And a "Restore existing wallet" option

### Additional Context
Here's the mock up for the android mobile wallet.
image

### PR DESCRIPTION
**Pull request by metaclips**
### Resolves

closes #2 

### What's new

Add wallet loading page which is viewed on UI startup also added wallet creation landing UI page

### Relevant screenshots or logs

Screenshot 2020-02-19 at 23 13 25
Screenshot 2020-02-01 at 07 07 13

### Additional context

- [x] Improve create/restore landing page

- [x] Improve Loading page that loads page while wallet is being initialized

- [x] Show wallet creation landing page only if wallet does not exist

### CODE REVIEW

File: ui/themes/materialplus/materialplus.go [hunk 1]
```diff
@@ -19,3 +19,11 @@ func Fill(gtx *layout.Context, col color.RGBA, x, y int) {
 	paint.PaintOp{Rect: dr}.Add(gtx.Ops)
 	gtx.Dimensions = layout.Dimensions{Size: d}
 }
+
+func RGB(c uint32) color.RGBA {
```
Comment (oluwandabira) [inline ui/themes/materialplus/materialplus.go]: comment

File: ui/themes/materialplus/materialplus.go [hunk 2]
```diff
@@ -19,3 +19,11 @@ func Fill(gtx *layout.Context, col color.RGBA, x, y int) {
...
+	return argb(0xff000000 | c)
+}
+
+func argb(c uint32) color.RGBA {
```
Comment (oluwandabira) [inline ui/themes/materialplus/materialplus.go]: comment

File: ui/widgets/imagebutton.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	"gioui.org/op/paint"
+	gioText "gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+
+	"github.com/raedahgroup/godcr-gio/ui/themes/materialplus"
+)
+
+type ImageButton struct {
```
Comment (oluwandabira) [inline ui/widgets/imagebutton.go]: comment

File: ui/window/window.go [hunk 4]
```diff
@@ -33,7 +34,7 @@ type Window struct {
 // than once.
 func CreateWindow(start string, wal *wallet.Wallet) (*Window, error) {
 	win := new(Window)
-	win.window = app.NewWindow(app.Title("GoDcr - decred wallet"))
+	win.window = app.NewWindow(app.Title("GoDcr - decred wallet"), app.Size(unit.Dp(550), unit.Dp(800)))
```
Comment (oluwandabira) [inline ui/window/window.go]: Don't set the size (or at least don't set it to portrait)

File: ui/widgets/imagebutton.go [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+		Src:      img,
+		VPadding: unit.Dp(16),
+		HPadding: unit.Dp(16),
+	}
+}
+
+func (b ImageButton) Layout(gtx *layout.Context, button *widget.Button, buttonTextSpace float32) {
+	hmin := gtx.Constraints.Width.Min
+	vmin := gtx.Constraints.Height.Min
```
Comment (oluwandabira) [inline ui/widgets/imagebutton.go]: Preferably `wmin` and `hmin` as that makes more sense

**Review by oluwandabira — CHANGES_REQUESTED** [review]

File: ui/widgets/imagebutton.go [hunk 6]
```diff
@@ -53,9 +55,10 @@ func NewImageButton(img *material.Image, text string) ImageButton {
 	}
 }
 
+// Layout renders the select instance on screen
```
Comment (oluwandabira) [inline ui/widgets/imagebutton.go]: select?

File: ui/themes/materialplus/materialplus.go [hunk 7]
```diff
@@ -20,10 +20,12 @@ func Fill(gtx *layout.Context, col color.RGBA, x, y int) {
 	gtx.Dimensions = layout.Dimensions{Size: d}
 }
 
+// RGB converts Hex to RGBA
 func RGB(c uint32) color.RGBA {
 	return argb(0xff000000 | c)
 }
 
+// RGB converts Hex to RGBA
 func argb(c uint32) color.RGBA {
```
Comment (oluwandabira) [inline ui/themes/materialplus/materialplus.go]: really?

File: ui/themes/materialplus/materialplus.go [hunk 8]
```diff
@@ -20,10 +20,12 @@ func Fill(gtx *layout.Context, col color.RGBA, x, y int) {
 	gtx.Dimensions = layout.Dimensions{Size: d}
 }
 
+// RGB converts Hex to RGBA
```
Comment (oluwandabira) [inline ui/themes/materialplus/materialplus.go]: could this be a little more descriptive

**Review by oluwandabira — CHANGES_REQUESTED** [review]

File: ui/page/landing.go [hunk 9]
```diff
@@ -18,90 +24,150 @@ const LandingID = "landing"
 // It should only be should shown if the app launches
 // and cannot find any wallets.
```
Comment (oluwandabira) [inline ui/page/landing.go]: Please remove. It's outdated now

File: ui/themes/materialplus/materialplus.go [hunk 10]
```diff
@@ -19,3 +19,13 @@ func Fill(gtx *layout.Context, col color.RGBA, x, y int) {
...
 	gtx.Dimensions = layout.Dimensions{Size: d}
 }
+
+// RGB converts Hex color code to color.RGBA
+func RGB(c uint32) color.RGBA {
+	return argb(0xff000000 | c)
+}
+
+// argb converts Hex color code to color.RGBA
+func argb(c uint32) color.RGBA {
```
Comment (oluwandabira) [inline ui/themes/materialplus/materialplus.go]: So they do the same thing?

**Review by oluwandabira — CHANGES_REQUESTED** [review]

File: ui/themes/materialplus/materialplus.go (hunk 10 above)
Comment (metaclips) [reply ui/themes/materialplus/materialplus.go]: argb sets alpha color code

**Comment by metaclips** [comment]
This is obsolete as button layout widget has been push upstream

**Pull request closed without merging**